src/main/webapp/app/exercises/quiz/manage/quiz-exercise-detail.component.ts

```diff
@@ -1208,6 +1209,20 @@ export class QuizExerciseDetailComponent implements OnInit, OnChanges, Component
         }
     }
 
+    /**
+     * Update adding existing questions from a file or course
+     */
+    setExistingQuestionSourceToCourse(setToCourse: boolean): void {
```

Comment:
Does it make sense to have a parameter for this and not just always set `showExistingQuestionsFromCourse` to true considering the other values are not parameterized too?

Reply:
The same function is used to set showExistingQuestionsFromCourse to false when user clicks "from a file" button at the html file. And making it toggle like this.showExistingQuestionsFromCourse = !this.showExistingQuestionsFromCourse; would lead to irregular behavior for default selection, when user clicks add existing question.

Reply:
Ah I see, thanks.